scripts/build.sh

```diff
@@ -127,12 +120,9 @@ fi
 
 export CGO_ENABLED=0
 
-${GOX:?command not found} \
-    -os="${XC_OS:-$ALL_XC_OS}" \
-    -arch="${XC_ARCH:-$ALL_XC_ARCH}" \
-    -osarch="${SKIPPED_OSARCH}" \
+go build \
     -ldflags "${GOLDFLAGS}" \
-    -output "pkg/{{.OS}}_{{.Arch}}/packer" \
+    -o "pkg/$(go env GOOS)_$(go env GOARCH)/packer" \
     .
```

Comment:
I think that this command was building a binary for all possible architectures handled by gox

Reply:
(ie, when running `make release`)

Reply:
Ah ok, so what should this be updated to then?

Reply:
Looking more into this, let me update with adding loop in build scripts through os/arch combinations.

Reply:
Update `build.sh` file to build the os and arch combinations like before. Let me know if this looks good and I'll update the `build.ps1` file (though that will be harder to test, I don't have a windows machine onhand).

Reply:
Ok now `build.sh` should be good to go, let me know if this looks good and I'll update `build.ps1`

Reply:
Updated `build.ps1` as well. I think it should be good (someone with a windows machine will have to confirm).